protocol/v2/ssv/runner/metrics/metrics.go

```diff
@@ -125,6 +138,7 @@ func (cm *ConsensusMetrics) EndPreConsensus() {
 func (cm *ConsensusMetrics) StartConsensus() {
 	if cm != nil {
 		cm.consensusStart = time.Now()
+		cm.metricsDutiesCreated.Inc()
```

Comment:
just saying that "fake duties" like validator registration or volunary exit won't be measured by this.

Also unless we know postconsensus was successful then we don't know if it was successful or not.

Since we use e2m anyhow for most duties we can simply change the title in grafana from "duties" to "consensus instances" to not confuse the user

Reply:
@GalRogozinski the question is which metric do we prefer? would you like to see all duties created or just consensus?